tools/packaging/windows/aws-otel-collector.wxs

```diff
@@ -55,7 +55,7 @@ Platform="x64"
     <ServiceInstall
 Id="Sevice"
 Name="AWSOTelCollector"
```

Comment:
What about the name? Is this a breaking change in any way if we were to change it?

Reply:
That's correct. it would be a breaking change and we shall track through an issue, I will create an issue to track this.